src/PixelFlags.cc

```diff
@@ -57,26 +59,70 @@ class FootprintBits : public afw::detection::FootprintFunctor<MaskedImageT> {
 private:
     typename MaskedImageT::Mask::Pixel _bits;
 };
-}  // end anonymous namespace
+typedef std::map<std::string, afw::table::Key<afw::table::Flag>> KeyMap;
```

Comment:
Please don't `typedef` this twice. You could make it public in the class definition and refer to `PixelFlagsAlgorithm::KeyMap`, perhaps.